### ISSUE TITLE
Auto Compressor Not work on server for pieces to ores

### ISSUE THREAD

**Issue by Teslachu** [NillerMedDild/Enigmatica2Expert#259]
So this is a very odd issue. The auto compressor from "ex compressum" is not working on a server, in regards to taking pieces (Like gold pieces, iron piece, etc) to ore form. Now it is important to note that the server was just updated to 1.5 this morning and the auto compressor was made after the update. So I have no clue if it was or would have worked before the update. There is no error associated with this bug. 

I did some further testing - The Auto compressor will work in private worlds for versions 1.4 and 1.5. It will also work on a back up of the server from 1.4, however, it will not work on the server. I am not sure if this is by design or what. I have double checked everything and retested this about 5 times, across 1.4 and 1.5, doing the exact same layout in all of them... 

TLDR: Works in private worlds / saves for 1.4 and 1.5, but not on a server. However, it also works on a backup of the server (when ran as a private world), but still will not work on the server, for taking to pieces to ore form. You can't even shift click (or input) the pieces in there at all.

**Comment by Teslachu** [comment]
Opps. Sorry!

**Comment by NillerMedDild** [comment]
No worries!
